### DISCUSSION TITLE
BLE sensors support - 'unofficial' version of BSB-LAN

### DISCUSSION THREAD

**Discussion by 1coderookie** [fredlcore/BSB-LAN#440] [Q&A] [answered]
_Here you can ask questions which may arise using the ['unofficial' version of BSB-LAN developed by @dukess which supports BLE sensors](https://github.com/dukess/BSB-LAN/tree/BLE-sensors) (ESP32 boards only!)._ 

Please read the belonging chapter of the BSB-LAN manual first.
English version: https://1coderookie.github.io/BSB-LPB-LAN_EN/chap07.html#744-xiaomi-mijia-ble-sensors
German version: https://1coderookie.github.io/BSB-LPB-LAN/kap07.html#744-xiaomi-mijia-ble-sensoren

**Comment by fredlcore** [comment, marked as answer]
Since there hasn't been any activity in the last 1.5 years, I'm closing this here. In any case, any kind of discussion would be better held at @dukess's repository linked above.
